src/main/java/Parser.java

```diff
@@ -21,11 +21,11 @@ public void parseCommand(String userInput) {
 
             tasks.list();
 
-        }  else if (userInput.length() > 3 && userInput.substring(0, 4).matches("done")){
+        } else if (userInput.length() > 3 && userInput.substring(0, 4).matches("done")) {
```

Comment:
Might want to consider utilizing `switch` statements to replace long `if-else` statements for readability